cfgmgr/buffermgrd.cpp

```diff
@@ -1,17 +1,17 @@
-#include <unistd.h>
-#include <getopt.h>
-#include <vector>
-#include <mutex>
+#include "buffermgr.h"
```

Comment:
Not sure why are these non-relevant changes. This was not in the previous commit. Could you please make sure you've only the new additions?

Reply:
Reverted. Sorry about that